metadata/location/utah_address_points

```diff
@@ -18,11 +18,11 @@ An address point represents a geographic location that has been assigned a US Po
 
 ### What is the dataset
 
-This dataset contains vector point data for over one million addresses in the State of Utah and includes information such as the street address and the type of unit the point represents. This dataset is a mapping-grade framework layer that receives regular updates. Maintaining this dataset is an ongoing effort between UGRC, counties, and local municipalities.
+This dataset contains vector point data for over one million addresses in the State of Utah and includes information such as the street address and the type of unit the point represents. This dataset is a mapping-grade framework layer that receives regular updates. Maintaining this dataset is an ongoing effort between UGRC and county-level data stewards.
```

Comment:
It's might be worth mentioning that address points are generally placed on the structure but can also be found on the driveway entrance or somewhere within the parcel. If a county provides an attribute noting where the point is we include it in our data.

Reply:
Great idea! I'll include that in the "what this data represents" section.